tests/test_feature_append_concat.py

```diff
@@ -104,6 +104,38 @@ def test_concat(test_df, reverse, iterable):
     npt.assert_array_equal(ts.iloc[2].values, ts.iloc[3].values)
 
 
+def test_concat_non_standard_index():
```

Comment:
I'd rather use "default" instead of "standard" because model-scenario is the default *index* when initializing an IamDataFrame.
```suggestion
def test_concat_non_default_index():
```